docs/schema/guidance/generating-statements.rst

```diff
@@ -0,0 +1,29 @@
+.. _generating-statements:
+
+Generating statements
+=====================
+
+Statements SHOULD be treated as immutable: once a Statement is published it SHOULD NOT be republished with the same statement identifier (``statementId``) and different property values. See :ref:`information-updates` for more information.
```

Comment:
Just to check my understanding - 'treated as immutable' rather than 'immutable' because there are very limited circumstances where a statement could be redacted?

Reply:
Yes, exactly.